src/services/platform.services.js

```diff
@@ -0,0 +1,47 @@
+import { PACKAGE_MANAGER } from '../config/app';
+const childProcess = window.require('child_process');
+const os = window.require('os');
+const path = window.require('path');
+const { remote } = window.require('electron');
+
+// Returns true if the OS is Windows
+export const isWin = (): boolean => /^win/.test(os.platform());
```

Comment:
Nit: Flow is pretty good about inferring types for return values, so the `: boolean` annotation isn't required, I _believe_ Flow knows that `.test` returns a bool.

If you (or Benny) prefers this, I don't mind it :) not requesting changes, just saying that for this and the other functions in the file, they can be omitted if you'd like.